roles/ffmpeg/tasks/ff_libraries.yml

```diff
@@ -26,6 +26,6 @@
   file: owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} state=directory path={{ ffmpeg_path }}
   register: ffmpeg_path={{ install_path }}/ffmpeg_sources
 
-- name: ff add ldconfig for /usr/local/lib
+- name: add ldconfig for /usr/local/lib
```

Comment:
This task name includes "ff" to distinguish it from the same task in the imagemagick install, so that the roles are completely modular and you can use either name to specify start-at-task when needed.